.github/CONTRIBUTING.md

```diff
@@ -33,3 +33,108 @@ This guide assumes a level of knowledge on how to use GitHub issues and pull req
 - Please keep your changes succinct in scope. Large pull requests with multiple changes to the codebase (i.e. multiple features) may be asked to be broken into multiple smaller pull requests.
 - Please give as much detail as is relevant about your approach to the solution in the pull request description.
 - When your code changes include any change to the visual UI, please include a screenshot of how your changes appear in the browser.
+
+## Start Coding
+
+Thissection is designed to help developers start coding in the project and understanding the basic concepts and components used.
+
+### Ember
+
+To understand the project you'll have to understand Ember, and it is advisable that you'll follow the tutorial: [Create your own app](https://guides.emberjs.com/v2.6.0/tutorial/ember-cli/) to get an understanding of how ember works and the basic folder structure.
```

Comment:
The tutorial guide url should be updated to:
https://guides.emberjs.com/v2.9.0/tutorial/ember-cli/